superset/security/manager.py

```diff
@@ -877,3 +878,39 @@ def assert_viz_permission(self, viz: "BaseViz") -> None:
         """
 
         self.assert_datasource_permission(viz.datasource)
+
+    def get_rls_filters(self, table):
+        """
+        Retrieves the appropriate row level security filters for the current user and the passed table.
+
+        :param table: The table to check against
+        :returns: A list of clause strings.
+        """
+        try:
+            roles = [role.id for role in g.user.roles]
```

Comment:
Lots of the same code as `get_rls_filters` - maybe stick this in a generalized method?

Reply:
but.. This is `get_rls_filters`, isn't it?

Reply:
My bad - meant `get_rls_ids` XD

Reply:
generalized